bot/teamsBot.ts

```diff
@@ -73,6 +74,26 @@ export class TeamsBot extends TeamsActivityHandler {
           }
           break;
         }
+        case "join office hours": {
+          if (this.activeQueue) {
+            if (this.activeQueue.checkQueue(context.activity.from.id)) {
+              context.sendActivity("You are already in queue.");
+            } else {
+              this.activeQueue.enqueueStudent(context.activity.from.id);
```

Comment:
It loos like a missing `await` on line 80 is causing the bot to crash whenever that branch is reached.

Reply:
Thanks for the catch!